INFO 2022-09-11 14:35:40,872 - Start time: 20220911T143540
WARNING 2022-09-11 14:35:49,370 - Exception encountered in fetch_trial_data. Traceback: Traceback (most recent call last):
  File "/global/cscratch1/sd/yazbeck/boa/boa/metaclasses.py", line 19, in wrapper
    return func(*args, **kwargs)
  File "/global/cscratch1/sd/yazbeck/boa/boa/wrapper_utils.py", line 52, in wrapper
    return func(*args, **kwargs)
  File "/global/cscratch1/sd/yazbeck/US_LA3_Wetl_C3Gra_Trans/elm_wrapper.py", line 422, in fetch_trial_data
    y_pred, y_true = get_model_obs(
  File "/global/cscratch1/sd/yazbeck/US_LA3_Wetl_C3Gra_Trans/elm_wrapper.py", line 257, in get_model_obs
    CurDayVar = CurDaydf.variables[ModVarName]
TypeError: unhashable type: 'list'

